docs/root/start/quick-start/run-envoy.rst

```diff
@@ -390,6 +390,22 @@ Some Envoy :ref:`filters and extensions <api-v3_config>` may also have additiona
 Envoy can be configured to log to :ref:`different formats <config_access_log>`, and to
 :ref:`different outputs <api-v3_config_accesslog>` in addition to files and ``stdout/err``.
 
+Envoy networking
+----------------
+
+By default Envoy can use both IPv4 and IPv6 networks. If your environment does not support IPv6 you should disable it.
```

Comment:
can we give this better spacing - as in the suggestion i posted

(a separating empty line is required to get a new line)

Reply:
done.